source/runbooks/revoking-user-access.html.md.erb

```diff
@@ -0,0 +1,169 @@
+
+Removing access for MP team members is documented [here](https://user-guide.modernisation-platform.service.justice.gov.uk/runbooks/removing-a-team-member.html)
+
+We also have a `testing-ci` user, defined in code [here](https://github.com/ministryofjustice/modernisation-platform/blob/ee861412ad30edb0c1ff763b6deeb2992d139f44/terraform/github/testing-ci.tf#L4) which is used for automated tests.
+It's access is limited to the testing-test account and essential core resources. To remove this user raise a PR to remove the user or edit the access of the associated policy.
+
+### MOJ Master Users
+
+There are a limited set of users who have access to the Root account ("MOJ Master") for the organisation.
+
+These users are defined in the [root account repo](https://github.com/ministryofjustice/aws-root-account/blob/b47c74781927ffbdb3cfba485d07b05ade7a23b9/management-account/terraform/iam-users.tf) 
+
+Any PRs raised to remove a users access would need to be approved by a member of [operations-engineering](https://github.com/orgs/ministryofjustice/teams/operations-engineering) or [aws-root-account-admin-team](https://github.com/orgs/ministryofjustice/teams/aws-root-account-admin-team). 
+
+## Teams
+
+GitHub Teams are linked to AWS environments via the `environment/*.json` files in the [MP repo](https://github.com/ministryofjustice/modernisation-platform/tree/main/environments).
+
+To remove a teams' access/association with an AWS environment:
+
+1. Edit the relevant `environment/*.json` file by removing the team name from the `github_slug` definition for the particular environment
+
+    e.g. `"github_slug": "modernisation-platform",` becomes  `"github_slug": "",`
+
+1. Note that only one change to a `environment/*.json` file can be made per PR/run due to current constraints around how the automation works.
+1. Raise a PR and once merged the scheduled baseline job will run to update the access accordingly.
+
+## Automation Users (roles/identities)
+
+###Member Roles
+
+####OIDC CI/CD Role
+
+To allow automated access to MP member environments we provide an optional “OIDC CI/CD” role - `modernisation-platform-oidc-cicd` which is 
+explained in more detail [here](https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/deploying-your-application.html#deploying-your-application)
+
+To remove this role for member accounts through code:
+
+1. Edit the relevant `environment/*.json` file by removing the team name from the `github-oidc-team-repositories` definition for the particular environment
+
+    e.g. `"github-oidc-team-repositories": ["modernisation-platform"]` becomes  `"github-oidc-team-repositories": [""]`
+
+1. Note that only one change to a `environment/*.json` file can be made per PR/run due to current constraints around how the automation works.
+1. Raise a PR and once merged the scheduled baseline job will run to update the access accordingly.
+
+
+####CircleCI Role
```

Comment:
Some of the headers need a space added in 👾 🙈

Reply:
Thanks @connormaglynn - sorted that now